### ISSUE TITLE
Improve the way to make breeding impossible for a Pokémon

### ISSUE THREAD

**Issue by AerunDev** [PokemonWorkshop/PokemonStudio#515]
As a User
I would like to improve the way to make breeding impossible for a Pokémon
So that the way to set that is easier and do not need any extra knowledge

## Things to know
For now, the user needs to set a baby Pokémon and set the breeding groups to Unknown to make sure a Pokémon cannot hatch eggs.
Could be better to set a baby to "None" to make sure this is easier to do it without extra knowledge.

## References
[See the Discord suggestion by Sukinaeくさ](https://discord.com/channels/143824995867557888/1378362834186141839)
